### PR TITLE
LPS-93338

### PR DESCRIPTION
**Pull request by oliv-yu**
New changes!

- Added data-testid's
- Mock DragSource, DropTarget
- More test cases for Results Ranking Form

### CODE REVIEW

File: modules/apps/portal-search/portal-search-ranking-web/test/js/components/ResultsRankingForm.es.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import ResultsRankingForm from 'components/ResultsRankingForm.es';
-import {cleanup, render, waitForElement} from 'react-testing-library';
+import {cleanup, fireEvent, render, waitForElement} from 'react-testing-library';
 import 'jest-dom/extend-expect';
 import '@babel/polyfill';
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-ranking-web/test/js/components/ResultsRankingForm.es.js]: What is `@babel/polyfill` needed for?

Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-ranking-web/test/js/components/ResultsRankingForm.es.js]: It's for the async function! Has to wait for the results to load.

https://github.com/babel/babel/issues/5085

**Comment by oliv-yu** [comment]
Thank you! Change sent. 👍

**Comment by thektan** [comment]
Sent https://github.com/BryanEngler/liferay-portal/pull/338

**Pull request closed without merging**